DSCResources/MSFT_xSQLAOGroupEnsure/MSFT_xSQLAOGroupEnsure.psm1

```diff
@@ -0,0 +1,351 @@
+$currentPath = Split-Path -Parent $MyInvocation.MyCommand.Path
```

Comment:
can you give a high level summary what this change is for and what scenarios it enables?